### ISSUE TITLE
Backbone 1.2.0

### ISSUE THREAD

**Issue by CupOfTea696** [paulmillr/exoskeleton#111]
Backbone recently release V1.2.0, any idea when you'll implement this into exoskeleton? Especially the new "update" event on Collections is useful.

http://backbonejs.org/#changelog

**Comment by akre54** [comment]
Yep, we'll merge over the changes in the next few days. If it takes longer than a week feel free to bug me or Paul.

**Comment by CupOfTea696** [comment]
That's awesome, thanks!

**Comment by alchemical** [comment]
Yup, I'd love to see this too. I'm just about to start rewriting a bunch of backbone stuff and was hoping to make the move to exoskeleton. Thanks.

**Comment by julienetie** [comment]
Great lib, looking forward to 1.2 & +

**Comment by akre54** [comment]
Hey y'all, I finished porting over the new code from Backbone. Several tests are failing at the moment and I don't have time to look into it. If anyone wants to take a crack they'd have my eternal gratitude: #112
